theme/js/script.js

```diff
@@ -0,0 +1,8 @@
+var swiper = new Swiper('.swiper-container', {
```

Comment:
Take this entire code snippet out, it shouldn't be here.

Reply:
it will be there for the testimonial

Reply:
Yes it will but you can take it out of this pull request because right now it doesn't make sense for it to be here.